pkg/graveler/graveler.proto

```diff
@@ -5,6 +5,13 @@ import "google/protobuf/timestamp.proto";
 
 package io.treeverse.lakefs.graveler;
 
+message RepositoryData {
```

Comment:
I noticed that you didn't add the UUID to the repository data. Did we decide not to go with this solution eventually?

Reply:
As we agreed:
* Added `func RepoPartition(repoID RepositoryID, _ Repository) string` to return the repository relevant partition key. Currently it returns the repo ID but eventually it will return some unique ID to identify the specific instance of the repository with the ID
* Decission on repository creation/deletion error handling and state-machine is extracted to another issue